### ISSUE TITLE
Подскажите пожалуйста, где найти dll для Quik8 x64

### ISSUE THREAD

**Issue by KirillV123** [Enfernuz/quik-lua-rpc#29]
error loading module 'lzmq' from file 'C:\QUIK\Include\lzmq.dll':
	%1 не является приложением Win32.

Установил [отсюда](http://zeromq.org/distro:microsoft-windows) библиотеку в надежде получить готовую dll
Но там целый список dll в папке bin:
![image](https://user-images.githubusercontent.com/52355665/61283801-5c92d280-a7c6-11e9-8e13-4a06677bf04c.png)

**Comment by saaremaa** [comment]
Как я понимаю quik-lua-rpc не поддерживает 64-bit'ную архитектуру? С Quik8 quik-lua-rpc запустить не получилось из-за несовместимости билиотек. Ошибка:
`error loading module 'lzmq' from file 'C:\PSBQuik\Include\lzmq.dll':
	%1 не является приложением Win32.
`

**Comment by KirillV123** [comment]
lzmq.dll я пересобрал под x64 использовал в качестве инструкции этот файл https://github.com/Enfernuz/quik-lua-rpc/blob/master/CONTRIBUTING.md
Но ошибка теперь совсем не понятная...
![image](https://user-images.githubusercontent.com/52355665/61375117-bcf94100-a8a6-11e9-9e0b-5566fa646436.png)
P.S. все файлы вроде в нужных местах лежат!

**Comment by Enfernuz** [comment]
Да, нужно разобраться со сборкой нужных зависимостей под 64-битную Lua. В данный момент занят личными делами, к сожалению, поэтому не могу быстро прореагировать. Ориентировочно, на этой неделе посмотрю. Если кто запилит до того момента -- пулл-реквесты приветствуются.

@KirillV123, это общая ошибка, когда QLua не может загрузить модуль по какой-то причине. Возможно, Ваш LuaRocks, когда собирает lzmq, линкует её с 32-битными .dll Lua (в CONTRIBUTING.md есть пункт, где советуется скачать 32-битную Lua для Windows и натравить LuaRocks на неё).

**Comment by Enfernuz** [comment]
Пересобрал все зависимости под Windows 10 x64 и Lua 5.1 x64. Пока не обновил репозиторий и документацию, можно просто воспользоваться готовыми библиотеками: 
[qlua-rpc-x64-dlls.zip](https://github.com/Enfernuz/quik-lua-rpc/files/3433549/qlua-rpc-x64-dlls.zip)
